### PR TITLE
DOC: Add citation materials

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add citation materials:
- Add a `how to cite` section to the `README`.
- Add the relevant BibTeX files.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Not sure why the last push force did not trigger the labeler actions, but will investigate if observed in other PRs whose push forces should actually change the labels.

**Comment by jhlegarreta** [comment]
@felixdumais I have just discovered that you have started using this other GitHub handle. I had requested a review for this PR. No pressure.

Edit: Is the new handle the one that you intend to use from now on?

File: README.md [hunk 1]
```diff
@@ -4,6 +4,32 @@ Tractography learning.
...
+    cite our work in tractography filtering, cite [FINTA] and [FIESTA].
+  - If you want to cite our work in tractography bundling, cite [CINTA] and
+    [FIESTA].
+    - If you use `tractolearn` to bundle streamlines using a k-nearest neighbor
+      label approach, cite [CINTA].
+    - If you use `tractolearn` to bundle streamlines using a thresholding
+      approach, cite [FINTA] and [FIESTA].
+  - If you use `tractolearn` for generative purposes or you want to cite our
+    work in generative models for tractography, cite [GESTA] and [FIESTA].
+  - If you use parts of `tractolearn` for othe purposes, please generally cite
```
Comment (felixdumais) [inline README.md]: Typo: `other`

File: README.md [hunk 2]
```diff
@@ -14,3 +40,9 @@ Patent #17/337,413. Pending.
 
 This software is distributed under a particular license. Please see the
 [*LICENSE*](LICENSE) file for details.
+
+
+[FINTA]: ./doc/bibtex/Legarreta21_-_MIA_-_FINTA.bib "Filtering in tractography using autoencoders (FINTA)"
+[CINTA]: ./doc/bibtex/Legarreta22_-_MICCAI-CDMRI_-_CINTA.bib "Clustering in Tractography Using Autoencoders (CINTA)"
+[GESTA]: ./doc/bibtex/Legarreta22_-_arXiv_-_GESTA.bib "Generative sampling in tractography using autoencoders (GESTA)"
+[FIESTA]: ./doc/bibtex/Dumais22_-_arXiv_-_FIESTA.bib "FIESTA Autoencoders for accurate fiber segmentation in tractography"
```
Comment (felixdumais) [inline README.md]: Typo: `FIESTA: `

**Review by felixdumais — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jhlegarreta) [reply README.md]: Not sure if I see the typo.

Comment (felixdumais) [reply README.md]: Well, the title I've put in arXiv takes a colon after the word FIESTA

File: README.md (hunk 1 above)
Comment (jhlegarreta) [reply README.md]: Fixed in  86eda20. Thanks.

File: README.md (hunk 2 above)
Comment (jhlegarreta) [reply README.md]: Ah, sorry, I should have paid attention. The bibtex was OK. Fixed in 98fa395.

**Pull request merged**